docs/section-6-contributing.md

```diff
@@ -18,7 +18,7 @@ To make changes to Tree-sitter, you should have:
 1. A C compiler, for compiling the core library and the generated parsers.
 2. A [Rust toolchain](https://rustup.rs/), for compiling the Rust bindings, the highlighting library, and the CLI.
```

Comment:
```suggestion
 2. A [Rust toolchain](https://rustup.rs/), for compiling the Rust bindings, the highlighting library, and the CLI.
- [ ]

Reply:
Hi @Sonyashines, could you clarify what change you'd like me to make? I'm not able to tell what you're looking for based on the diff.